keras/utils/layer_utils.py

```diff
@@ -334,7 +386,9 @@ def print_layer(layer, nested_level=0, is_nested_last=False):
                '|' * nested_level)
 
   layers = model.layers
-  for layer in layers:
+  for i, layer in enumerate(layers):
```

Comment:
Shouldn't we just get `layer = model.layers[layer_range[0], layer_range[1]]`?

Reply:
It'd be great. With last commit i changed the function output to allow this list slicing (adding 1 to the second layer_range index). I also had to change vis_utils.py accordingly. Thanks!